x-pack/plugins/apm/public/components/shared/charts/duration_distribution_chart/total_doc_count_label.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { css } from '@emotion/react';
+import { useEuiTheme } from '@elastic/eui';
+import { EuiText } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { ProcessorEvent } from '../../../../../common/processor_event';
+
+interface Props {
+  eventType: ProcessorEvent.transaction | ProcessorEvent.span;
+  totalDocCount?: number;
+}
+
+export function TotalDocCountLabel({ eventType, totalDocCount }: Props) {
+  const { euiTheme } = useEuiTheme();
+
+  if (!totalDocCount) {
+    return null;
+  }
+
+  return (
+    <EuiText
+      size="s"
+      css={css`
+        border-left: ${euiTheme.border.thin};
+        padding-left: ${euiTheme.size.s};
+      `}
+    >
+      {eventType === ProcessorEvent.transaction
+        ? i18n.translate(
+            'xpack.apm.durationDistributionChart.totalTransactionsCount',
+            {
+              defaultMessage: '{totalDocCount} total transactions',
```

Comment:
Don't you need to add a plural handle here as well? Something like?!
```suggestion
              defaultMessage: '{totalDocCount, plural, one {total transaction} other {total transactions}}',
```

Reply:
Good spot, added in https://github.com/elastic/kibana/pull/136960/commits/0382609c0b1f03eb70426323c205021b8c4f6883